src/base/COBRARequisitesFullfilled.m

```diff
@@ -0,0 +1,275 @@
+function [solversToUse] = COBRARequisitesFullfilled(varargin)
```

Comment:
Wouldn't it make sense to name this function `setTestRequirements` or `defineTestRequirements` and move it to `src/base/install`? I also think there should be a status returned.

Reply:
`testTestRequirements` would make sense, or `testRequirementsAndGetSolversForTest()`.
Because it actively tests for the requirements, and errors if they are not fullfilled.

About the returned status:
If the function succeeds, the Requirements are fullfilled. If it fails (i.e. if it throws an error they are not).
We need some way to distinguish between missing requirements and test failures. this function at the moment allows to add :
```
solvers = functioncall('Properties',values)
```
to a test to obtain the available and used requested solvers
if we would have a status, we would need the following in a test:
```
global ERROR_ID_GLOBAL
[status,missingRequirements] = testRequirements('param',value)
assert(status, ERROR_ID_GLOBAL, missingRequirements)
usedSolvers = getSolversForTest('param',value)
```
I find the former much easier for users.